userdocs/profiles.dev/docs/intro.md

```diff
@@ -4,8 +4,128 @@ sidebar_position: 1
 
 # Introduction
 
-What is a profile?
+Welcome to Profiles: _The_ GitOps Native Package Manager :sparkles:
 
-What problem is the profiles mechanism trying to solve?
+:::danger _(highway to the)_ Danger Zone
+Profiles is still highly experimental and likely to experience sudden and significant
+API change as we continue to develop based on incoming feedback.
 
-etc
+Please always refer to the docs and release notes, and alert us to any issues
+or inconsistencies.
+
+Thank you!
+:::
+
+## What are Profiles?
+
+In short, a profile is a Kubernetes package. Much as you would expect from another package
+manager (`brew`, `apt`, etc), the Profiles mechanism allows Kubernetes operators
+to manage their clusters in the same way they manage their host systems. Available
+packages are listed somewhere searchable; they are versioned; they are tested and verified;
+they are installable by way of a single simple command; they are updatable and upgradable.
```

Comment:
simple command;

`single` is redundant as a and ... and command already suggest it's singular.